_playwright-tests/UI/RedHatRepo.spec.ts

```diff
@@ -0,0 +1,17 @@
+import { describe } from 'node:test';
+import { test, expect } from '@playwright/test';
+import { navigateToRepositories } from './helpers/navHelpers';
+import { closePopupsIfExist } from './helpers/helpers';
+
+describe('Red Hat Repositories', () => {
+  test('Red Hat repository exists and is valid', async ({ page }) => {
+    await navigateToRepositories(page);
+    await closePopupsIfExist(page);
+    const redHatToggle = page.getByTestId('redhat-repositories-toggle');
+    await redHatToggle.click();
+
+    await expect(page.getByText('Red Hat Ansible Engine 2 for RHEL 8 x86_64 (RPMs)')).toBeVisible();
+    await expect(page.getByText('Valid')).toBeVisible();
+    await expect(page.getByText('Last snapshot')).toBeVisible();
```

Comment:
These two are too generic. They only check that any repo in the list has `Valid` and `Last snapshot` visible. It would fail if there would be more than one repo (because it is enforced that only one element is being targeted).

I guess in our local setup we have only this one repo, so it works. But if we would run this test against stage it would not work. I think for writing cleaner test it would be better to target one specific repo and check on it, that it contains these values. 

Storing the first selector and then filtering might be easy and readable :) https://playwright.dev/docs/locators#filtering-locators

Reply:
oh yep i meant to add this in! just pushed it up in its current state yesterday to test the gh action :)